src/CommunityToolkit.Maui.Maps/AppHostBuilderExtensions.shared.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace CommunityToolkit.Maui.Maps;
+
+/// <summary>
+/// Maps Extensions for <see cref="MauiAppBuilder"/>
+/// </summary>
+public static class AppHostBuilderExtensions
+{
+	/// <summary>
+	/// Initializes the .NET MAUI Community Toolkit Maps Library
+	/// </summary>
+	/// <param name="builder"><see cref="MauiAppBuilder"/> generated by <see cref="MauiApp"/> </param>
+	/// <param name="key"></param>
+	/// <returns><see cref="MauiAppBuilder"/></returns>
+	public static MauiAppBuilder UseMauiCommunityToolkitMaps(this MauiAppBuilder builder, string key)
+	{
+		builder.UseMauiMaps();
+		builder.ConfigureMauiHandlers(handlers =>
+		{
+#if WINDOWS
+			CommunityToolkit.Maui.Maps.Handlers.MapHandlerWindows.MapsKey = key;
```

Comment:
Should we throw an exception when the key is not there? Or add that to the settings we have in place for that?

Reply:
we throw an exception in the handler: throw new InvalidOperationException("You need to specify a Bing Maps Key"); We added it there to ensure the user cant bypass initialization with this extension method

Reply:
Overlooked that one, thanks!